docs/source/conf.py

```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'GraphQL Compiler'
+copyright = '2019-Present Kensho Technologies, LLC'
```

Comment:
From the html file, this gets displayed as "2019-Present Kensho Technologies, LLC.", (with a dot at the end). If we add a dot after LLC here it will look like "2019-Present Kensho Technologies, LLC.."

Reply:
Can we make the `Present` bit not be capitalized?

Reply:
Add this comment into the file itself, please. Otherwise, when we look at this in 6 months, we'll try to "fix" it again.